### PR TITLE
LPS-159648 Fix Poshi Automation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-159648

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro [hunk 1]
```diff
@@ -1086,9 +1086,11 @@ definition {
...
 			DisplayPageTemplate.clickPencilItem(item = "Edit Webcontent Title");
 
-			WebContent.addCP(
-				webContentContent = "Webcontent Content Edit",
-				webContentTitle = "Webcontent Title Edit");
+			PortletEntry.inputTitle(title = "Webcontent Title Edit");
+
+			CKEditor.addContentInWebContent(
+				content = "Webcontent Content Edit",
+				fieldLabel = "content");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/Upgrade.macro:1093]: Use WebContent.addWithStructureCP
webContentRichText = "content"

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! I've updated the PR with the macro you requested. Could you have a look at?

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**